.github/workflows/main.yml

```diff
@@ -20,10 +20,15 @@ jobs:
               run: |
                   yarn install
                   yarn checkSize
+            - name: Clean directory
+              run: |
+                  git clean -ffdx --exclude /tmp
```

Comment:
Oh, didn't know you could do double `-f`

Niiiice....

Reply:
I copied this from the `actions/checkout` [docs](https://github.com/actions/checkout#usage). Since all we wanted to do was add the `--exclude`, i just disabled clean and copied their code.